client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -87,7 +92,12 @@ export const useUrlQueryParams = ({
       const filterValue = getFilterValue(urlQuery, filter.key);
       if (!filterValue) return prev;
 
-      prev[filter.key] = getFilterEntry(filter, filterValue);
+      const [key, nested_key] = filter.key.split(NESTED_SPLIT_CHAR);
```

Comment:
`nested_key` -> `nestedKey`

Do you have the eslint ext installed?

Reply:
Installed~ thanks Clemens